product_docs/docs/biganimal/release/getting_started/preparing_cloud_account/preparing_gcp/index.mdx

```diff
@@ -0,0 +1,133 @@
+
+!!! Note
+    Before proceeding, see [Understanding requirements in Google Cloud](understanding_quotas_in_gcp) for details on planning for your clusters' requirements and resource limits in Google Cloud.
+
+
+## Required roles
+
+Ensure you have at least the following combined roles:
+- roles/iam.serviceAccountCreator
+- roles/iam.serviceAccountKeyAdmin
+- roles/iam.roleAdmin
+- roles/resourcemanager.projectIamAdmin
+- roles/compute.viewer
+
+Or an equivalent single role such as:
+- roles/owner
+
+## Required APIs and services
+
+Ensure the following Google Cloud APIs are enabled:
+
+- autoscaling.googleapis.com               
+- cloudapis.googleapis.com                 
+- cloudresourcemanager.googleapis.com      
+- compute.googleapis.com                   
+- container.googleapis.com                 
+- iam.googleapis.com                       
+- iamcredentials.googleapis.com            
+- run.googleapis.com                       
+- secretmanager.googleapis.com             
+- storage.googleapis.com                   
+- vpcaccess.googleapis.com     
+
+
+## Check Google Cloud resource limits for running BigAnimal
+
+EDB provides a shell script, called [biganimal-csp-preflight](https://github.com/EnterpriseDB/cloud-utilities/blob/main/gcp/biganimal-csp-preflight), which checks whether requirements and resource limits are met in your Google Cloud account based on the clusters you plan to deploy.
+
+1.  Open [Google Cloud Shell](https://cloud.google.com/shell) in your browser.
+2.  From Google Cloud Shell, run the following command:
+
+    ```shell
+    curl -sL https://raw.githubusercontent.com/EnterpriseDB/cloud-utilities/main/gcp/biganimal-csp-preflight | bash -s <GCP-account-ID> <region> [options]
+    ```
+    The required arguments are:
+
+    | Argument | Description |
+    | -------- | ----------- |
+    | &lt;account-id> | Google Cloud account ID of your BigAnimal deployment. |
+    | &lt;region> | Google Cloud region where your clusters are being deployed. See [Supported regions](../../overview/03a_region_support/#supported-google-cloud-regions) for a list of possible regions. |
+
+    Possible options are:
+
+    | Options | Description |
+    | ------- | ----------- |
+    | `-h` or `--help`| Displays the command help. |
+    | `-i` or `--instance-type` | Google Cloud instance type for the BigAnimal cluster. The help command provides a list of possible VM instance types. Choose the instance type that best suits your application and workload. Choose an instance type in the memory optimized M1, M2, or M3 series for large data sets. Choose from the compute-optimized C2 or C2D series for compute-bound applications. Choose from the accelerator-optimized A2 or A2 ultra for massively parallelized Compute Unified Device Architecture (CUDA) workloads. Choose from the general purpose M5 or M6I series if you don't require memory or compute optimization.|
```

Comment:
we won't support C2D or any accelerator optimized instance types at launch. only the instance types that have a "Yes" in column K here: https://docs.google.com/spreadsheets/d/1WWBAEgJjVWwSIKBoU8z4rIt3x1p8pXMMn7JkahWtYxQ/edit#gid=1228767107
and the general purpose instance families we support at MVP will be E2, N2, and N2D - not m5 or m6i, those are AWS volumes